rpa-manager/modules/ROOT/pages/botmanagement-manage.adoc

```diff
@@ -1,11 +1,36 @@
 = Managing Bots
 
-Bots can be licensed to use secure sessions. Some properties, like name, service times, and phase permissions can be edited. Deinstalled bots can be removed from RPA Manager.
+Newly installed bots connect to RPA Manager automatically.
+
+As a bot administrator, you can manage the following properties of a bot:
+
+* Name and describe a bot.
+* Assign service times to a bot to determine their office hours.
+* Restrict bots to project phases
+* Set the maximum number of sessions a bot may use.
+
+If a bot is troubling you, you can do the following:
```

Comment:
"troubling" is troubling me :) 

Can you be more specific? If a bot isn't working properly?